common/openssl-compat.c

```diff
@@ -87,6 +88,16 @@ void ECDSA_SIG_get0(const ECDSA_SIG *sig, const BIGNUM **pr,
   }
 }
 
+int BN_bn2binpad(const BIGNUM *a, unsigned char *to, int tolen) {
+  int n = BN_num_bytes(a);
+  if (n < 0 || n > tolen)
```

Comment:
braces on those two ifs would be nice

Reply:
Fixed in 3ce940dca5a0a563b3bc836119e7eba43b6182c4